### ISSUE TITLE
Appropriate GPU memory

### ISSUE THREAD

**Issue by TLB-MISS** [bytedance/MVDream-threestudio#16]
How much GPU memory is needed for MVDream's soft-shading, batch8, 512 resolution setting (i.e. the paper setting)?
You said A100 GPU in git, but it's not clear because there are two types of A100, 40GB and 80GB.

Thanks

**Comment by murlock1000** [comment]
For reference, I was able to train a model without soft-shading for 5000 iterations with an RTX 3080 Ti (12GB).

The settings I had to change were:
batch_size: [8,4] -> [4,4]

**Comment by TLB-MISS** [comment]
@murlock1000 
Thanks. But.. If you change n_view from 4 to 1, won't the meaning of 'multi view' disappear?

**Comment by murlock1000** [comment]
You're right. I've changed it back to 4. This did not affect the memory usage but increased the run time. Interestingly, the 3D model still converged without multi-view support, but it took significantly more iterations to converge to a 3D model and did not provide highly detailed features.

**Comment by yanjk3** [comment]
A100-80G is required for running the soft-shading configuration.
